docs/nodes/widgets/ui-template.md

```diff
@@ -37,29 +37,43 @@ UI Template will parse different tags and render them into Dashboard. The availa
 Any variables that you want to render into your `<template />` are done so in one of two ways:
 
 ::: v-pre
-- **Attribute Binding** - Use `:` to bind a variable to an attribute. For example: 
+- **Attribute Binding** - Use `:` to bind a variable to an attribute. Anything inside the `""` here is treated as JavaScript, for example: 
 
 ```html
 <p :class="msg.payload">Hello World</p>
 ````
 
-- **Text Interpolation** - Use `{{ }}` to interpolate a variable into the text of an element. For example:
+or, if we want to modify `msg.payload` before using it:
```

Comment:
Somewhat open to interpretation - it reads like "this is how you can modify msg.payload" to me.

```suggestion
or, if you want to use `msg.payload` as part of the value, you can do this:
```